components/Board/Profiles.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import Profile from "./Profile";
+
+const arr = [
+	{
+		Id: "a",
```

Comment:
Do not include the id, simply because if we are to change around the order, then you have to manually change the ID as well. Instead when we map over this array, we can grab the index from the map function directly and apply that to the key prop. You can remove them for all of the objects.